SignalShareExtension/utils/ShareAppExtensionContext.m

```diff
@@ -29,8 +37,6 @@ - (instancetype)initWithRootViewController:(UIViewController *)rootViewControlle
 
     _rootViewController = rootViewController;
 
-    OWSSingletonAssert();
```

Comment:
`OWSSingletonAssert` is turning out to be really useful.  I'd like to do something similar for Swift.  I have an idea...